docs/source/install.rst

```diff
@@ -181,6 +181,44 @@ running the LiberTEM GUI. In other environments, like Jupyter notebooks, you
 need to restart the Python interpreter to make sure the new version is used,
 for example by restarting the ipython kernel.
 
+.. _`airgapped`:
+
+Air-gapped installation
+-----------------------
+
+Many microscope control computers are not connected to the internet, which means
+that the usual installation methods don't work. It is not straightforward to
+package a Python application into a self-contained executable or installer for
+Windows, see also :issue:`39`. Furthermore, relocating Python environments, for
+example with `conda-pack <https://conda.github.io/conda-pack/>`_ doesn't always
+work reliably. In order to install LiberTEM on an air-gapped machine, you need a
+computer with the same operating system, architecture and Python version as a
+host system where you can prepare all required packages for the target system.
+
+On the host
+^^^^^^^^^^^
+
+* Create a folder where you collect all required files, for example :code:`wheels`.
+* If necessary, download and install a recent version of Python supported by LiberTEM from
+  https://www.python.org/downloads/. This installer works without internet access.
```

Comment:
Maybe we should explicitly say that this version of course has to be compatible with the operating system version you will install on, which could be older on air-gapped systems (as we depend on Python 3.9, that means currently Windows 8+ IIUC)